libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.java

```diff
@@ -550,18 +549,6 @@ public JSONObject getJSONCredentials() {
    }
 
     /**
-     * Exception thrown if initial web page load fails.
-     */
-    public static class HybridAppLoadException extends RuntimeException {
```

Comment:
This is probably from back in the day when we had the bootstrap page for auth in hybrid apps.